changelog.json

```diff
@@ -1,5 +1,9 @@
 {
   "releases": {
+    "2.3.2-beta5": [
```

Comment:
I left out `Disable creating issues from desktop on repos that have issues disabled - #9207` because, while it's a user-facing change, the user impact isn't that pronounced. That said, I'm open to including it if folks thing it's worth adding.